<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorDeviceWizardSshPage</class>
 <widget class="QWizardPage" name="Mer::Internal::MerEmulatorDeviceWizardSshPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>656</width>
        <height>319</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="userGroupBox">
            <property name="title">
             <string>User &quot;nemo&quot;</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="userNameLabel">
               <property name="text">
                <string>Username:</string>
               </property>
               <property name="buddy">
                <cstring>userLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="userLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="userSshKeyLabel">
               <property name="text">
                <string>SSH key:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="userSshKeyLabelEdit">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="userAuthorizedFolderLabelEdit">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="userAuthorizedFolderLabel">
               <property name="text">
                <string>Authorized keys:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="userSshCheckBox">
               <property name="text">
                <string>Generate new SSH keys</string>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="rootGroupBox">
            <property name="title">
             <string>User &quot;root&quot;</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="rootNameLabel">
               <property name="text">
                <string>Username:</string>
               </property>
               <property name="buddy">
                <cstring>userLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="rootLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="rootSshKeyLabel">
               <property name="text">
                <string>SSH key:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="rootSshKeyLabelEdit">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="rootAuthorizedFolderLabelEdit">
               <property name="text">
                <string>none</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="rootAuthorizedFolderLabel">
               <property name="text">
                <string>Authorized keys:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="rootSshCheckBox">
               <property name="text">
                <string>Generate new SSH keys</string>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
